incubator/couchbase/templates/svc.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata: 
+  name: {{ template "fullname" . }}
+  labels: 
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec: 
+  ports:
+  - name: {{ template "fullname" . }}
+    port: 8091
+  selector: 
+    app: {{ template "fullname" . }}
+    couchbaserole: "master"
+  type: LoadBalancer 
```

Comment:
Make this configurable and default it to ClusterIP. Especially since the username and password are easily guessable and static.